initial/src/main/java/com/example/resources/NumbersResource.java

```diff
@@ -0,0 +1,16 @@
+package com.example.resources;
+
+import com.example.services.NumberService;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class NumbersResource {
+    public static final String URL_PREFIX="/numbers";
+
+    @RequestMapping(URL_PREFIX + "/random")
+    public int getRandom() {
+        NumberService ns = new NumberService();
```

Comment:
NOT AN ISSUE, this comment is for my own note. We will soon go through @Component and make sure all service classes are initiated during service startup.